ci/prepare.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: "Prepare the environment for running stf"
+  hosts: controller
+  tasks:
+    - name: "Update pip"
+      ansible.builtin.pip:
+        name: pip
+        state: latest
+        extra_args: "-U"
+
+    - name: "Install pre-reqs from pip"
+      ansible.builtin.pip:
+        requirements: "../build/stf-run-ci/requirements.txt"
+        chdir: "{{ playbook_dir }}"
+        state: present
+
+    - name: "Install ansible collections"
+      community.general.ansible_galaxy_install:
+        type: collection
+        name: "{{ item }}"
+      with_items:
+        - "kubernetes.core:2.3.2"
+        - "community.general:6.2.0"
```

Comment:
Is there a better place to control these versions than hard coding them into the play?

Reply:
It could be put into a requirements.yml file under {{ sto_dir }}/build/stf-run-ci. 

The format of that MAY depend on the version of ansible/galaxy

Reply:
Won't block approving/merging this without the change, but I always get itchy with hard coded versions especially as we have dependabot which will propose updating library versions.